README.md

```diff
@@ -5,7 +5,19 @@ The medic-android application is a thin wrapper to load the [CHT Core Framework]
 
 # Release notes
 
-### 0.7.3
+## 0.8.0
+
+### Changes
+
+- [improvement] [#134](https://github.com/medic/medic-android/issues/134) New connection errors UX:
+  - The improvements only apply to _Webview_ flavors.
+  - It also applies when the app migrates to Webview from a XWalk installation.
+- [improvement] [#163](https://github.com/medic/medic-android/issues/163) New UX of Crosswalk to Webview migration:
```

Comment:
You have the issue numbers and descriptions mismatched here.

134 = `Improve UX of crosswalk to webview migration`

163 = `Improve connection errors UX, especially for the XWalk-Webview migration process`

Reply:
:facepalm: fixed.